tests/conftest.py

```diff
@@ -325,7 +307,7 @@ def fixture_parameters(
         ),
     ]
 
-    if _CHEM_INSTALLED:
```

Comment:
there seems to be a difference between ONNX optionals and others (like CHEM), the latter being inly imported based on the `_optional` info but onnx not? Can you comment on this?

Reply:
can you elaborate? Not sure what you mean... Perhaps I simply forgot to apply a change somewhere?

Reply:
thres no equivalent a la `if ONNX_INSTALLED` which must mean optionality there is differently handles compared to the rest, just wanted to understand that

Reply:
ok i see now where the difference comes from, but I dont think any change is needed, I expect validation at instantiation and there are prob not many usecases outside of testing/mocking to use objects that require additional deps without having the additional deps installed